README.md

```diff
@@ -17,9 +17,39 @@ Then alter your homeserver configuration, adding to your `modules` configuration
 modules:
   - module: threepid_checker.ThreepidChecker
     config:
-      # TODO: Complete this section with an example for your module
+      # The URL to send requests to when checking if a 3PID can be associated to an account.
+      # See below for more information.
+      # Required.
+      url: https://foo/bar
+
+      # If true, the module will only send requests to the URL as part of registering a
+      # new user. This means no request will be sent to the URL when associating a 3PID with
+      # an already existing account.
+      # Optional, defaults to false.
+      only_check_at_registration: false
 ```
 
+The configured URL will be hit by a `GET` HTTP request, with 2 parameters to qualify the 3PID:
+
+* `medium`: The 3PID's medium (`email` for an email address, `msisdn` for a phone number)
+* `address`: The 3PID's address
+
+The server at that URL is expected to respond with a JSON object that contains the following keys:
+
+* `hs` (string): Required. The name of the homeserver the 3PID is allowed to be associated
+                 on. The 3PID will be denied if this is absent from the response's body.
```

Comment:
What does "name" mean here? Is it the server part of an MXID? From below it seems to be "server name", but it could be nice to give an example (or point to the exact config variable name in the homeserver config?)